swarmdjango/swarm_backend/urls.py

```diff
@@ -14,8 +14,54 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
+from rest_framework import routers
+
+# Import all the view sets
+from core.views import userViewSet
+from core.views import adminViewSet
+from core.views import changeViewSet
+from core.views import commentViewSet
+from core.views import developerViewSet
+from core.views import devPersonalPageViewSet
+from core.views import entryViewSet
+from core.views import headingViewSet
+from core.views import personalPageViewSet
+from core.views import photoGalleryViewSet
+from core.views import sideBarViewSet
+from core.views import sponsorPersonalPageViewSet
+from core.views import sponsorViewSet
+from core.views import wikiViewSet
+from core.views import logViewSet
+from core.views import robotViewSet
+from core.views import runViewSet
+
+router = routers.DefaultRouter()
+
+# Route view sets via the router
+router.register('user', userViewSet.UserViewSet, basename='user')
+router.register('admin2', adminViewSet.AdminViewSet, basename='admin2')
```

Comment:
I might have missed something, but why is this being referenced as admin2?

Reply:
Temporary usage, as the route 'admin' exists already for django's built-in admin app

Reply:
Is this something that we should delete moving forward?

Reply:
More likely it will be renamed. The admin part refers to our admin data model, however Django includes an admin url+site, so currently we can't map the route right to 'admin'.

Reply:
Good to know